src/Bootstrap/ButtonGroup.elm

```diff
@@ -57,12 +72,30 @@ type GroupItem msg
     = GroupItem (Html.Html msg)
 
 
-{-| Opaque type representing a button or link button, for composing button groups
+{-| Opaque type representing a button, for composing button groups
 -}
 type ButtonItem msg
     = ButtonItem (Html.Html msg)
 
 
+{-| Opaque type representing a link button, for composing button groups
+-}
+type LinkButtonItem msg
```

Comment:
Sorry, but this one passed me by the last time. 

Does this have to be a separate type ? I mean mixing link buttons and ordinary buttons (sorry) shouldn't really be a problem and can't think of a really good reason to not allow that ?